tests/integration/docusaurus/connecting_to_your_data/how_to_configure_a_runtimedataconnector.py

```diff
@@ -77,9 +95,12 @@
     ]
 )
 
+# <snippet name="tests/integration/docusaurus/connecting_to_your_data/how_to_configure_a_runtimedataconnector.py path">
 path = "<PATH TO YOUR DATA HERE>"
```

Comment:
This one isn't rendering properly for some reason
![Screen Shot 2023-02-22 at 1 23 31 PM](https://user-images.githubusercontent.com/25670697/220762165-539c536f-334a-41b1-8c21-7336fefaa798.png)